### ISSUE TITLE
For timezone selections, give options

### ISSUE THREAD

**Issue by dylanmtaylor** [archlinux/archinstall#173]
The current flow is confusing for timezone selection, and the valid options should be provided. US/Eastern isn't exactly an intuitive thing to enter if you don't know what options are available. Maybe do a menu like with keyboard layouts.

**Comment by Torxed** [comment]
As mentioned in the [locale issue](https://github.com/archlinux/archinstall/issues/147#issuecomment-813053613), there are 348 available options, and I don't think they will all fit on the screen. So chunking them up becomes important.

As discussed there, perhaps splitting up on continents first, or do something like:
```python
import archinstall
regions = set([x.split(b'/')[0] for x in archinstall.sys_command("timedatectl list-timezones --no-pager")])
```
And present only those options in a first step.
```
{b'', b'Europe', b'Atlantic', b'Antarctica', b'UTC\r', b'Indian', b'Africa', b'America', b'Australia', b'Pacific', b'Asia'}
```

It would add two steps instead of one, by first show all the regions, and then show all the cities.
If this is an acceptable solution it would partially solve #147 as well.

Bottom line: I agree that this step needs improvement as well as the locale chooser, it's just a matter of keeping it simple and informative.

**Comment by dylanmtaylor** [comment]
> It would add two steps instead of one, by first show all the regions, and then show all the cities.
> If this is an acceptable solution it would partially solve #147 as well.
> 
> Bottom line: I agree that this step needs improvement as well as the locale chooser, it's just a matter of keeping it simple and informative.

I feel like segmenting it by region -> specific time zone is probably the best solution. It's how some installs for other distros work as well.

**Comment by Torxed** [comment]
There's a new menu system being worked on, that will have a listing of all available time-zones. Just dropping a ping here to know that I've tagged that PR to this issue.
